modules/library/render/src/main/java/org/geotools/renderer/lite/gridcoverage2d/ColorMapNode.java

```diff
@@ -21,6 +21,7 @@
 import it.geosolutions.jaiext.classifier.LinearColorMapElement;
 import it.geosolutions.jaiext.piecewise.Domain1D;
 import it.geosolutions.jaiext.range.NoDataContainer;
+import it.geosolutions.rendered.viewer.RenderedImageBrowser;
```

Comment:
I don't believe this one is appropriate... btw, where is it hosted now? jai-ext?